web/yo/app/scripts/controllers/reportgenerator.js

```diff
@@ -295,14 +295,55 @@ angular.module('oncokbApp')
 
                 DatabaseConnector.getReviewedData($scope.reviewedData[$scope.evidenceType].evidenceTypes).then(function(response) {
                     if ($scope.evidenceType === 'geneType') {
-                        _.each(response, function(item) {
-                            $scope.reviewedData.geneType.body.push({
-                                gene: item.hugoSymbol,
-                                oncogene: item.oncogene,
-                                tsg: item.tsg
+                        var variantLookupBody = _.map(response, function(item) {
```

Comment:
```
_.map(_.uniq(_.pluck(response, 'hugoSymbol')), function(hugoSymbol){
    return {
        hugoSymbol: hugoSymbol
    }
})
```

Reply:
_.map can take object directly and access as key-value pairs, it is unnecessary to form an array first

Reply:
The purpose is getting the unique hugoSymbol

Reply:
I thought there is no duplication in the genes list returned by the genes API. You think there might be?

Reply:
... oh ok. You nested genes call in here as well. If that's the case, no. There is no dup.